examples/platforms/k32w/jn5189/Makefile.am

```
@@ -0,0 +1,141 @@
+    -I$(top_srcdir)/third_party/nxp                                                                                  \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6                                                                        \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/devices/JN5189                                                         \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/middleware/mbedtls/port/ksdk                                           \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/components/serial_manager                                              \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/components/uart                                                        \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/devices/JN5189/drivers                                                 \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/devices/JN5189/utilities/debug_console                                 \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/devices/JN5189/utilities/str                                           \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/boards/jn5189dk6/wireless_examples/openthread/enablement               \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/CMSIS/Include                                                          \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/middleware/wireless/ieee-802.15.4/uMac/Include                         \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/middleware/wireless/framework/XCVR/DK6/Build/Include                   \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/middleware/wireless/framework/XCVR/DK6                                 \
+    -I$(top_srcdir)/third_party/nxp/JN5189DK6/middleware/wireless/framework/Common                                   \
+    -Wno-unknown-pragmas                                                                                             \
+    -Wno-sign-compare                                                                                                \
+    -Wno-unused-function                                                                                             \
+    -Wno-unused-parameter                                                                                            \
+    -Wno-empty-body                                                                                                  \
+    $(NULL)
+
+libopenthread_jn5189_sdk_a_CPPFLAGS                                                                                = \
+    $(LIB_FLAGS)                                                                                                     \
+    $(NULL)
+
+libopenthread_jn5189_plat_a_CPPFLAGS                                                                               = \
+    $(LIB_FLAGS)                                                                                                     \
+	$(NULL)
```

Use spaces instead of tab.
```suggestion
    $(NULL)
```

---

Solved.